caikit/runtime/service_generation/data_stream_source.py

```diff
@@ -216,65 +204,92 @@ def make_data_stream_source(data_element_type: Type) -> Type[DataBase]:
     if data_element_type not in _DATA_STREAM_SOURCE_TYPES:
         cls_name = _make_data_stream_source_type_name(data_element_type)
         log.debug("Creating DataStreamSource[%s] -> %s", data_element_type, cls_name)
-        element_type = (
-            data_element_type.get_proto_class().DESCRIPTOR
-            if isinstance(data_element_type, type)
-            and issubclass(data_element_type, DataBase)
-            else _NATIVE_TYPE_TO_JTD[data_element_type]
+
+        # Set up the "sub classes." In python, this is the same as creating a
```

Comment:
I can't say that this is really _prettier_ than the nasty `schema` stuff, but it's at least consistent now, so this change enabled fully removing the JTD-style of `@dataobject`.